.github/workflows/tests.yml

```diff
@@ -1,6 +1,6 @@
 name: Tests
 
-on: [ push, pull_request ]
+on: [ push, pull_request_target ]
```

Comment:
You didn't change this on the python SDK. What's the reason you changed it here? I don't quite get what it does from reading the docs, but maybe that's my limited experience with github actions https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target

Reply:
Yeah so in the python sdk there is mechanism o implicitly use the `GITHUB_TOKEN` and specify github actions as the service to send reports to coveralls.

In the java sdk however we use the coveralls repo token instead. So when we have a push/pr from a fork (recall the [issue](https://bynder.slack.com/archives/D01BNLKP708/p1609334804000700) last month that you brought to my notice)
```
Error:  Failed to execute goal org.eluder.coveralls:coveralls-maven-plugin:4.3.0:report (default-cli) on project bynder-java-sdk: Build error: Either repository token or service with job id must be defined -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
```

This happens because the repo token isn't shared across the fork. 


What `pull_request_target` does is, it enables that across-fork sharing to enable us to run coveralls coverage action even when we have a pr across forks.

Reply:
With `pull_request`:
This is the maven coveralls command check [here](https://github.com/Arpit-Sharma-USC/bynder-java-sdk/runs/1784727492?check_suite_focus=true)
```
mvn coveralls:report \
  --no-transfer-progress \
  -D repoToken=
```

Notice the error:

`Error:  Failed to execute goal org.eluder.coveralls:coveralls-maven-plugin:4.3.0:report (default-cli) on project bynder-java-sdk: Build error: Either repository token or service with job id must be defined`

Reply:
With `pull_request_target`:
This is the command, check [here](https://github.com/Arpit-Sharma-USC/bynder-java-sdk/runs/1784727369?check_suite_focus=true)
 ```
mvn coveralls:report \
  --no-transfer-progress \
  -D repoToken=***
```

It is able to fetch the repo token for my repo here